crates/rust-analyzer/src/config.rs

```diff
@@ -2032,7 +2029,7 @@ impl Config {
 
     pub fn client_commands(&self) -> ClientCommandsConfig {
         let commands = self.commands();
-        let force = commands.is_none() && *self.lens_forceCustomCommands();
+        let force = commands.is_none();
```

Comment:
Bit isn't this worse than the current status quo?  Currently,  the whole thing can be disabled with lens_forceCustomCommands=false.  With this PR, if the client does not support anything, the server will assume it supports everything.  Right?

Reply:
Sorry, I was thinking that this was the intended behavior, then the intended fix would be to remove "force" all together, right?

Reply:
Yeah, I _think_ we should get rid of it. The Code extension does set it properly in https://github.com/rust-lang/rust-analyzer/blob/4afe0d5393cdedde58881295752fe68acb3148ae/editors/code/src/client.ts#L344.

Reply:
We should remove the `force` behavior fully

Reply:
I forgot to reply, I pushed a new commit that should have the correct code now